arkouda/pdarrayclass.py

```diff
@@ -809,3 +837,189 @@ def std(pda, ddof=0):
     unbiased estimate of the standard deviation per se.
     """
     return np.sqrt(var(pda, ddof=ddof))
+
+def mink(pda, k):
+    """
+    Find the `k` minimum values of an array.
+
+    Returns the smallest `k` values of an array, sorted
+
+    Parameters
+    ----------
+    pda : pdarray
+        Input array.
+    k : integer
+        The desired count of minimum values to be returned by the output.
+
+    Returns
+    -------
+    pdarray
+        The minimum `k` values from pda
+
+    Notes
+    -----
+    This call is equivalent in value to:
+    
+        a[ak.argsort(a)[:k]]
+    
+    and generally outperforms this operation.
+
+    This reduction will see a significant drop in performance as `k` grows
+    beyond a certain value. This value is system dependent, but generally
+    about a `k` of 5 million is where performance degredation has been observed.
+    
+    Examples
+    --------
+    >>> A = ak.array([10,5,1,3,7,2,9,0])
+    >>> ak.mink(A, 3)
+    array([0, 1, 2])
+    """
+    if isinstance(pda, pdarray):
+        if k == 0:
+            return []
+        if pda.size == 0:
+            raise TypeError("must be a non-empty pdarray {} of type int".format(pda))
```

Comment:
message says "...of type int" I thought we were supporting int64 and float64 dtypes?